src/mcpdotnet/Server/McpToolAttribute.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace McpDotNet.Server;
+
+/// <summary>
+/// Attribute to mark a method as an MCP tool.
+/// </summary>
+[AttributeUsage(AttributeTargets.Method)]
+public class McpToolAttribute : Attribute
+{
+    /// The name of the tool. If not provided, the class name will be used.  
```

Comment:
Missing `<summary></summary>`?

And the class name will be used rather than the method name?